scripts/copyright/UpdateAllCopyrightHeaders.sh

```diff
@@ -44,7 +44,7 @@ GetExitCodeText(){
             echo "    Error: Too many arguments"
             ;;
         "8")
-            echo "    Warning: There are files that need their copyright headers updated"
+            echo "    Warning: There are files that need their copyright headers updated. (Please run UpdateAllCopyrightHeaders.sh --update)"
```

Comment:
That will be nice to remember that I have to use the `--update` flag to make the changes.  Sometimes I forget and have to look it up.